README.md

```diff
@@ -124,6 +125,20 @@ Design, prototype, and collaborate all in the browser—with Figma.
 | Platform | macOS, Windows, web |
 | URL | https://www.figma.com |
 
+### Taskade
+
+Team productivity, reimagined.
```

Comment:
Let's remove the marketing aspect of things 😄 To a PM reading this document, "reimagined" doesn't mean much. I would emphasize your value proposition and functionality supported.